modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -194,7 +190,7 @@ export async function getAllObjectFolders() {
 	);
 }
 
-export async function getFolderByERC(folderERC: string) {
+export async function getObjectFolderByERC(folderERC: string) {
```

Comment:
@igor-franca Please, sort it in alphabetical order. Remember to regen the .d.ts file.